fem/bilininteg.hpp

```diff
@@ -2671,11 +2671,22 @@ class ElasticityIntegrator : public BilinearFormIntegrator
 /** Integrator for the DG form:
     alpha < rho_u (u.n) {v},[w] > + beta < rho_u |u.n| [v],[w] >,
     where v and w are the trial and test variables, respectively, and rho/u are
-    given scalar/vector coefficients. The vector coefficient, u, is assumed to
-    be continuous across the faces and when given the scalar coefficient, rho,
-    is assumed to be discontinuous. The integrator uses the upwind value of rho,
-    rho_u, which is value from the side into which the vector coefficient, u,
-    points. */
+    given scalar/vector coefficients. {v} represents the average value of v on
+    the face and [v] is the jump such that {v}=(v1+v2)/2 and [v]=(v1-v2) for the
+    face between elements 1 and 2. For boundary elements, v2=0. The vector
+    coefficient, u, is assumed to be continuous across the faces and when given
+    the scalar coefficient, rho, is assumed to be discontinuous. The integrator
+    uses the upwind value of rho, rho_u, which is value from the side into which
+    the vector coefficient, u, points.
+
+    When used with ConvectionIntegrator, the transpose of this form with
+    alpha=-1.0, beta=0.5 can be used to implement the upwind flux and outflow
+    boundary conditions, see ex9 and ex9p.
+
+    When combined with the transpose of ConvectionIntegrator integrator, the
+    coefficients alpha=1.0, beta=0.5 can be used to implement the upwind flux
+    and outflow boundary conditions in conservative form.
```

Comment:
```suggestion
    Another use case for this integrator is to discretize the operator -div(u v)
    with a DG formulation. The resulting formulation is conservative and
    consists of the transpose of the ConvectionIntegrator (with coefficient u,
    and parameter alpha = 1) plus the DGTraceIntegrator (with coefficient u, and
    parameters alpha = -1, beta = -1/2 to use the upwind face flux).
```
I guess the current comment talks about outflow b.c. because it describes the discretization of `div(u v)` where inflow becomes outflow. I use `-div(u v)` as an example because it is the r.h.s. of the simple conservation law `dv/dt = -div(u v)` -- the conservation of mass law (v = density, u = velocity).